lib/logstash/outputs/elasticsearch/http_client.rb

```diff
@@ -5,6 +5,8 @@
 require 'logstash/outputs/elasticsearch/http_client/manticore_adapter'
 
 module LogStash; module Outputs; class ElasticSearch;
+  TARGET_BULK_BYTES = 20 * 1024 * 1024
```

Comment:
Put a comment here something maybe explaining why we made this a constant instead of a tunable? My thought is that I want to avoid having users yelling at us about a "bad decision" making this not configurable (even though we agreed to make this not a configuration setting)